src/node_perf.cc

```diff
@@ -291,6 +291,22 @@ void MarkBootstrapComplete(const FunctionCallbackInfo<Value>& args) {
       performance::NODE_PERFORMANCE_MILESTONE_BOOTSTRAP_COMPLETE);
 }
 
+static double PerformanceNowImpl() {
+  return static_cast<double>(uv_hrtime() - performance_process_start) /
```

Comment:
Would you mind updating this to be relative to `env->time_origin()`? Even though they are the same thing at the moment in Node.js, the web spec states that every [environment](https://html.spec.whatwg.org/#environment-settings-object) has its own time origin value, so the time origin of a web worker is different from the main context.

Reply:
I think when we do give each worker its own time origin, we can just change `performance_process_start` to a thread local and initialize it in the workers? There's probably no need to get the entire environment for this.

Reply:
Well, yeah, this is not a blocker. Please feel free to dismiss this since they are effectively the same at the moment.